libvirt/libvirt_test.go

```diff
@@ -96,12 +107,82 @@ func TestLibirt(t *testing.T) {
 		expectedType := make(map[string]int64)
 		So(cpu, ShouldHaveSameTypeAs, expectedType)
 	})
+	Convey("Get Network Statistics without network interface neither path", t, func() {
+		conn, err := libvirt.NewVirConnection("test:///default")
+		defer conn.CloseConnection()
+		So(err, ShouldBeNil)
+		domains, err := GetInstances(conn)
+		network, err := GetNetworkStatistics(domains[0])
+		So(err, ShouldNotBeNil)
+		emptyMap := make(map[string]libvirt.VirDomainInterfaceStats)
+		So(network, ShouldResemble, emptyMap)
+	})
+	Convey("Get Network Statistics with one path", t, func() {
+		conn, err := libvirt.NewVirConnection("test:///default")
+		So(err, ShouldBeNil)
+		_, err = conn.DomainCreateXMLFromFile("./test_domain_multiple_interfaces.xml", 0)
+		So(err, ShouldBeNil)
+		defer conn.CloseConnection()
+		name := "instance-00000031"
+		domain, err := GetInstanceByDomainName(conn, name)
+		So(err, ShouldBeNil)
+		network, err := GetNetworkStatistics(domain, "tap88709cbd-90")
+		So(network["tap88709cbd-90"].RxBytes, ShouldNotBeNil)
```

Comment:
Is it possible to check if network parameters have specific values instead of checking if they are not Nil?